### DISCUSSION TITLE
Resize charts by mouse dragging of divider line between charts

### DISCUSSION THREAD

**Discussion by publine2** [highfestiva/finplot#209] [Q&A]
I have more than one chart in same window(like fplt.create_plot(..., rows=2, ...)).
Is it possible to resize charts by mouse dragging of divider line between charts, like in TradingView? Will be very appreciated for code example.

**Comment by highfestiva** [comment]
Nope, they need to have a predefined when creating the window, see https://github.com/highfestiva/finplot/wiki/Snippets#scaling-plot-heights.

**Reply by publine2** [reply]
Just a suggestion: what you think about docking system of pyqtgraph? That is making charts sizable, closeable, movable, attachable, detachable, .....
just a raw example:

from pyqtgraph.dockarea import *

area = DockArea()
d1 = Dock("AAPL", size=(500, 300))
d2 = Dock("RSI",size=(500,300), closable=True)
area.addDock(d1, 'bottom')
area.addDock(d2, 'bottom', d1)
w1 = ...plot AAPL prices here
w2 = ...calculate RSI.....
d1.addWidget(w1)
d2.addWidget(w2)

**Reply by highfestiva** [reply]
This kinda thing should be doable, see examples/embed.py.

**Reply by publine2** [reply]
embed.py - that is not the same, it's just showing how to use finplot as a QT Widget.  Actually, didn't get why it's called 'TradingView wannabe' - what is from TradingView there? It's not looking like TradingView and not behave itself as TradingView.

**Reply by publine2** [reply]
well... of course we can use QT to write whenever we want, but what I'm speaking about is already implemented in pyqtgraph.dockarea, so will be much ease just use that instead of writing that from scratch.

**Reply by ATILIUS-REGULUS** [reply]
Howdy, I am new to finplot and I am impressed by the speed compared to bokeh and plotly.
It would be a great feature, if the heigths and widths of a chart could be resized intereactively. 
This is a feature, I was looking for at bokeh before, but couldn't find it.
Is there any chance, that it will be included in finplot in the near future?

**Reply by highfestiva** [reply]
Yes, an enhancement is currently posted on that topic.
